text/0000-embroider-v2-package-format.md

```diff
@@ -487,7 +487,7 @@ export default implementation;
 // ===============
 // Or compile down to this if OwnConfig contains { useNewVersion: false }
 let implementation
-implementation = importSync("./new-component");
+implementation = importSync("./old-component");
```

Comment:
This one is not a bug, it's intentional to show that the output is different if the `useNewVersion` flag is false.

Reply:
Oh wait, sorry, you're right, I had it backwards.